app/src/main/res/layout/upload_depicts_fragment.xml

```diff
@@ -103,29 +103,29 @@
         android:background="@color/divider_grey" />
 
     <Button
-        android:id="@+id/depicts_next"
-        style="@style/Widget.AppCompat.Button.Borderless"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginEnd="24dp"
-        android:layout_marginRight="24dp"
-        android:layout_marginBottom="24dp"
-        android:layout_alignParentEnd="true"
-        android:layout_alignParentRight="true"
-        android:layout_alignParentBottom="true"
-        android:text="@string/next" />
+      android:id="@+id/depicts_next"
+      style="@style/Widget.AppCompat.Button"
```

Comment:
I would think omitting this is equivalent so this line can be deleted

Reply:
@siamsot does removing this style line produce the same results?

Reply:
Indeed it does so I removed the line